File: app/src/main/java/com/neome/api/meta/base/dto/EntVdPipelineVarMap.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdPipelineVar;

import java.util.Map;

@SuppressWarnings("unused")
public class EntVdPipelineVarMap extends StudioBase
{
  public MetaIdPipelineVar[] keys;

  public Map<MetaIdPipelineVar, EntVdPipelineVar> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdPipelineVarMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdPipelineVar
import com.neome.api.meta.base.Types.MetaIdPipelineVar
import com.neome.api.meta.base.dto.StudioBase

interface EntVdPipelineVarMap : StudioBase
{
  val keys: List<MetaIdPipelineVar>
  val map: Map<MetaIdPipelineVar, EntVdPipelineVar>
}